OpenEphys.Onix/OpenEphys.Onix/ConfigureHeadstage64ElectricalStimulator.cs

```diff
@@ -26,6 +26,10 @@ static class Headstage64ElectricalStimulator
     {
         public const int ID = 4;
 
+        // NB: could be read from REZ but these are costant
```

Comment:
```suggestion
        // NB: could be read from REZ but these are constant
```

Small typo. I guess these bits are a minor refactoring of the Headstage64 electrical stim. Usually these would go in a separate PR, but in the interests of time we can go with it.